stdlib/runtime/numbers.gr

```diff
@@ -0,0 +1,1336 @@
+/* compilation-mode: runtime */
+
+import Tags from "runtime/unsafe/tags"
+import Errors from "runtime/unsafe/errors"
+
+import WasmI32 from "runtime/unsafe/wasmi32"
+import WasmI64 from "runtime/unsafe/wasmi64"
+import WasmF32 from "runtime/unsafe/wasmf32"
+import WasmF64 from "runtime/unsafe/wasmf64"
+
+import foreign wasm throwError: (WasmI32, WasmI32, WasmI32) -> a from "grainRuntime"
```

Comment:
Yes, `throwError` lets you throw a specific error, like `DivisionByZero`—purely a runtime thing. `fail` calls `throwError` with `Errors._ERR_FAILURE`. There'll be a PR that ports that to Grain too.

It's not a primitive because we needed JS to throw an actual JS error. (In wasm, we'll print and trap or print and exit(-1).)